packages/common/backup.yml

```
@@ -184,9 +184,14 @@ actions:
         DS_folder=/share/datastore
         mirror_folder=~/share_mirror
 
+        if [[ "$(grep DX_VESTION /.jelenv | sed 's/.*=\(8\.1\)\..*/match/')" = "8.1" ]]; then
```

Typo here, should be DX_VERSION

---

```suggestion
        if (grep -q '^DX_VERSION=8.1.' /.jelenv); then
```
looks better and I guess more functional than substite `8.1.*` to `match` and then test if if equal to `8.1` :raised_eyebrow:

---

indeed, don't know why first thought about the complicated way...